modules/converting-machine-api-infra-machines.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id="converting-machine-api-infra-machines_{context}"]
+= Converting infrastructure machines created using the Machine API
+
+[IMPORTANT]
+====
+Creating infra machines using the Machine API is now deprecated.
+====
+
+Infrastructure machines created using the Machine API had both the `worker` and `infra` labels applied. This duel label required you to taint the node so that compute workloads were not scheduled on it. Managing infra workloads using taints is no longer recommended. Machines created using the Machine API must be converted to use only the `infra` label and have a custom machine config pool manage it.
+
+.Procedure
+
+. List the infra nodes available in the cluster:
++
+[source,terminal]
+----
+$ oc get nodes -l node-role.kubernetes.io/infra
+----
++
+.Example output
+[source,terminal]
++
+----
+NAME                           STATUS   ROLES          AGE   VERSION
+ip-10-0-205-158.ec2.internal   Ready    worker,infra   64m   v1.20.0+551f7b2.20.0
+----
+
+. Remove the `worker` label from the node you want to use as an infra node:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/worker-
+----
+
+. Create a dedicated machine config pool to manage your infra nodes.
```

Comment:
Another link to the section describing how to move infrastructure workload to the new nodes could be helpful: https://docs.openshift.com/container-platform/4.7/post_installation_configuration/cluster-tasks.html#moving-resources-to-infrastructure-machinesets

Suggestion: "You can now move infrastructure workload to the infrastructure nodes that you created above." with the link above.